File: api/src/main/java/me/kvdpxne/dtm/arena/map/ArenaMapException.java
package me.kvdpxne.dtm.arena.map;

import me.kvdpxne.dtm.arena.ArenaException;

public class ArenaMapException extends ArenaException {
  public ArenaMapException(String message, String code) {
    super(message, code);
  }

  public ArenaMapException(String message, Throwable cause, String code) {
    super(message, cause, code);
  }
}

File: src/main/kotlin/me/kvdpxne/dtm/arena/map/ArenaMapException.kt
package me.kvdpxne.dtm.arena.map

import me.kvdpxne.dtm.arena.ArenaException

/**
 * @since 0.1.0
 */
open class ArenaMapException(
  // @formatter:off
  message: String     = "",
  cause  : Throwable? = null
  // @formatter:on
) : ArenaException(message, cause) {

  companion object {

    /**
     * Serial version UID for serialization compatibility.
     *
     * @since 0.1.0
     */
    @Suppress("ConstPropertyName")
    private const val serialVersionUID: Long = 1450370118356522309L
  }
}